rohit/widgets/src/components/Accordion.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+
+const Accordion = ({ items }) => {
+  
+  const [activeIndex, setActiveIndex] = useState(null)
+  
+  const onTitleClicked = (index) => {
+    setActiveIndex(index)
+  }
+
+  const renderedItems = items.map( (item, index)=> {
```

Comment:
Let's pay more attention to code formatting. There should be spaces around operators/symbols.